.github/workflows/scheduled-publishing.yml

```diff
@@ -0,0 +1,50 @@
+name: "Scheduled publishing"
+run-name: "Publish to internal - Scheduled"
+
+on:
+    schedule:
+    -   cron: "0 0 * * SUN"
+
+jobs:
+    publish:
+        strategy:
+            matrix:
+                package:
+                -   "dbt-athena"
+                -   "dbt-bigquery"
+                -   "dbt-postgres"
+                -   "dbt-redshift"
+                -   "dbt-snowflake"
+                -   "dbt-spark"
+            fail-fast: false
+        uses: ./.github/workflows/publish-internal.yml
+        with:
+            package: ${{ matrix.package }}
+        secrets: inherit
+
+    open-lockfile-pr:
+        runs-on: ${{ vars.DEFAULT_RUNNER }}
+        steps:
```

Comment:
You'll want a needs so this doesn't run until the publish job finishes

Reply:
Good catch; I updated this.